src/components/modals/WalletListModal.js

```diff
@@ -76,11 +80,23 @@ export function WalletListModal(props: Props) {
     excludeCurrencyCodes
   } = props
 
+  // #region Constants
+
   const account = useSelector(state => state.core.account)
   const [searching, setSearching] = useState(false)
   const [searchText, setSearchText] = useState('')
+  const [linkedBankAccounts, setLinkedBankAccounts] = useState()
+  useAsyncEffect(async () => {
+    const wyreClient = await makeWyreClient({ account })
+    if (wyreClient.isAccountSetup) {
+      setLinkedBankAccounts(await wyreClient.getCryptoPaymentMethods())
+    }
+  })
+
+  // #endregion Constants
+
+  // #region Init - Upgrade deprecated props
```

Comment:
I'm getting use to this region thing. Though, notice how an "end" is usually followed by the beginning of another region? I'd like to see an instances where this isn't the case because it seems like "endregion" is redundant (infer the end of one region by the start of another region. Of course, unless we nest regions (but that seems a bit overboard; how many region levels can we have?).

Reply:
We can nest regions, and #endregions are required for the folding to work at all. A #region followed by another #region is considered the start of a nested region.

Not sure how far we can nest (don't see why there would be a limit), but at that point it's probably prudent to start just making new constants/functions.